### ISSUE TITLE
Server keeps loading when using // import { } from '#imports'; in the server/api directory

### ISSUE THREAD

**Issue by Rock070** [nuxt/nuxt#22153]
### Environment

Node Version: v16.20.0
Nuxt Version: 3.6.2

### Reproduction

https://stackblitz.com/edit/nuxt-starter-5s58h6?file=server%2Fapi%2Fcount.ts

### Describe the bug

When including the commented line `// import { } from '#imports';` in files under the `server/api` directory, the server remains in a continuous loading state. Even though the line is commented out, the issue persists. However, upon removing the mentioned line of code, the loading issue no longer occurs.

```ts
// import { } from '#imports';

export default defineEventHandler((event) => {
  return 2;
});
```
### Additional context

<img width="410" alt="image" src="https://github.com/nuxt/nuxt/assets/53036805/d95b1a29-f2b4-434d-b3fd-cbc10fdd601c">

**Comment by danielroe** [comment]
cc: @antfu

**Comment by antfu** [comment]
Fixed in `unimport@v3.1.2`. Before the next Nuxt release, try to remove your lock and do a clean install to have the latest unimport. Thanks.
